packages/grid/x-data-grid/src/hooks/features/keyboard/useGridKeyboardNavigation.ts

```diff
@@ -29,12 +30,26 @@ export const useGridKeyboardNavigation = (
   const logger = useGridLogger(apiRef, 'useGridKeyboardNavigation');
   const currentPage = useGridVisibleRows(apiRef, props);
 
+  /**
+   * @colIndex index of the column to focus
+   * @rowIndex index of the row to focus
+   * @closestColResolution Which closest column cell to focus when cell has `colSpan`.
```

Comment:
"closestColResolution"?

Reply:
Is naming confusing? Maybe `closestCellResolution` is better?

Reply:
I was not referring to the name but this JSDoc comment is wrong. It should start with "@param...". About the name maybe `relativeColumnToUse`? We don't have the specific column but only in which direction to look for a column. Also good to avoid abbreviations.

Reply:
Thanks, I didn't even notice that jsdoc is wrong
I've decided to name it `closestColumnToUse`